README.md

```diff
@@ -45,97 +46,28 @@ we aim to minimize our dependencies, but still rely on ones such as
 - certbot (ssl certificate tool)
 - cmake and build-essential (build tools for c/c++)
 
-as well as some others.
-```sh
-# python3.9 is often not available natively,
-# but we can rely on deadsnakes to provide it.
-# https://github.com/deadsnakes/python3.9
-sudo add-apt-repository -y ppa:deadsnakes
-
-# install required programs for running bancho.py
-sudo apt install -y python3.9-dev python3.9-distutils \
-                    cmake build-essential \
-                    mysql-server redis-server \
-                    nginx certbot
-
-# install python's package manager, pip
-# it's used to install python-specific dependencies
-wget https://bootstrap.pypa.io/get-pip.py
-python3.9 get-pip.py && rm get-pip.py
-
-# make sure pip and setuptools are up to date
-python3.9 -m pip install -U pip setuptools
-
-# install bancho.py's python-specific dependencies
-python3.9 -m pip install -r requirements.txt
-```
-
-## creating a database for bancho.py
-you will need to create a database for bancho.py to store persistent data.
-
-the server uses this database to store metadata & logs, such as user accounts
-and stats, beatmaps and beatmapsets, chat channels, tourney mappools and more.
+these dependencies are all retrieved by and contained within docker containers.
```

Comment:
i wouldn’t straight up remove regular setup requirements; people should have the option for docker or manual setup